framework-plugins/lisk-framework-dashboard-plugin/src/ui/components/Input/TextAreaInput.tsx

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright © 2021 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+import * as React from 'react';
+import styles from './Input.module.scss';
+
+interface Props {
+	placeholder?: string;
+	value?: string;
+	onChange?: (val: string) => void;
+	size?: 's' | 'm' | 'l';
+}
+
+const TextAreaInput: React.FC<Props> = props => {
+	const { placeholder } = props;
+	const size = props.size ?? 'm';
+	const [value, updateValue] = React.useState(props.value);
+
+	const handleOnChange = (val: string) => {
+		updateValue(val);
+	};
+	const onChange = props.onChange ?? handleOnChange;
```

Comment:
This code will copy `value` prop to local state and if someone provided `onChange` prop, the local state will not be updated hence text change will not be seen in the text area.